src/definitions/modules/dropdown.js

```diff
@@ -3854,6 +3862,8 @@ $.fn.dropdown.settings = {
   ignoreDiacritics       : false,      // match results also if they contain diacritics of the same base character (for example searching for "a" will also match "á" or "â" or "à", etc...)
   hideDividers           : false,      // Whether to hide any divider elements (specified in selector.divider) that are sibling to any items when searched (set to true will hide all dividers, set to 'empty' will hide them when they are not followed by a visible item)
 
+  title                  : 'search',       // input title attribute
```

Comment:
Sorry for being a grinch again :grin:, but if the think about the A11Y scope a bit further and want to avoid a breaking change in the future, we should name this `aria.searchLabel`, because `title` (better `aria.label `) should be reserved for the whole dropdown element. What about a more future proof setting:
```javascript
aria:{
  title:'dropdown', 
  searchTitle:'search' 
} 
```

Reply:
Did the typo myself.. Of course the settings should read label instead of title